### PR TITLE
Syndetroit

### PR DESCRIPTION
**Pull request by terezi4real**
## About the PR

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added Syndetroit, a custom edit of the most popular and beloved map in the game.

### CODE REVIEW

File: Resources/Prototypes/Maps/syndetroit.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+- type: gameMap
+  id: Syndetroit
+  mapName: 'Syndetroit'
+  mapPath: /Maps/syndetroit.yml
+  minPlayers: 30
+  maxPlayers: 60
+  stations:
+    Barratry:
```
Comment (NakataRin) [inline Resources/Prototypes/Maps/syndetroit.yml:8]: Try changing this to `syndetroitstation` or something. The only way I see to fix it. If you have no other ideas, it's worth a shot, in my opinion.

**Comment by NakataRin** [comment]
Someone forgot to add spawnpoint for Passenger, Librarian, Musician 💀

**Comment by terezi4real** [comment]
> Someone forgot to add spawnpoint for Passenger, Librarian, Musician 💀

they were removed when i changed their areas for the factory, forgot to add em back.
who are you again?

**Comment by terezi4real** [comment]
woohoo looks like all bugs ironed out

**Comment by terezi4real** [comment]
and now checks fail after minor map changes :godo:

**Comment by sTiKyt** [comment]
> > Someone forgot to add spawnpoint for Passenger, Librarian, Musician 💀
> 
> they were removed when i changed their areas for the factory, forgot to add em back. who are you again?

They are someone helpful :D

**Comment by sTiKyt** [comment]
> and now checks fail after minor map changes :godo:

Your minor change looks like you just removed file contents and put map into it, for some reason xD

image

**Comment by terezi4real** [comment]
oops lol i did that shit at like 8am

**Pull request merged**